lib/schema.json

```diff
@@ -2,18 +2,25 @@
   "type": "object",
   "properties": {
     "keysRoot": {
+      "description": "Absolute or relative path for module ids to be relative to",
       "type": "string",
       "minLength": 1
     },
     "moduleEnding": {
+      "description": "Ending of module file names, for exampe: \".translations.json\"",
       "type": "string",
       "minLength": 1,
       "pattern": "\\.json$"
     },
     "dictionaryPattern": {
+      "description": "Absolute or relative path to the dictionary files with [locale_code] in file name.",
       "type": "string",
       "minLength": 1,
       "pattern": "\\[locale_code\\].*\\.json$"
+    },
+    "emitFile": {
```

Comment:
Not sure about the setting name. To `emit` a file in webpack usually means to produce something like an asset in the build folder. This setting actually changes a **source file** 

Don't have a good suggestion. Maybe `patchDictionaries`?

Reply:
-> `updateDictionaries`?
it is an asset in a way. not sure

Reply:
will keep it for now, it's an asset, existing in manifest.json and so on